README.md

```diff
@@ -41,102 +40,107 @@ pre-commit install
 pytest
 mkdocs build
 ```
+</details>
 
-***
-
-## :rocket: Usage
+## Usage :rocket: 
 
 Without any configuration needed, you can execute
 ```
-poetry kedro run --pipeline "prepare"
+kedro run --pipeline prepare
 ```
 followed by
 ```
-poetry kedro run
+kedro run
 ```
-(or omit poetry if you're using classical venvs) and a default pipeline should run.
+and a default pipeline should run. In this and following examples the leading `poetry run` is omitted for simplicity.
 
 Note that is required to always run the "prepare" pipeline in advance to any actual processing pipeline.
 This is because of the current implementation relies on dynamically created nodes that are depending on the configuration and therefore requiring two separate pipeline executions.
 
 In summary, the following pipelines exist:
 - "prepare" : generates all possible combinations of configurations based on the current parameter set
-- "measure" : performs the actual time measurement by executing experiments for each of the previously generated configuration
-- "ctmeasure" : continous a previous time measurement
-- "visualize" : gathers all the experiment results and generates some nice plots
+- `measure` : performs the actual time measurement by executing experiments for each of the previously generated configuration with the ability to parallelize processing
+- `ctmeasure` : continous a previous time measurement
+- `combine` : gathers all the results from the `measure` pipeline and combines them into a single output dataset
+- `visualize` : takes the combined experiment results and generates some nice plots
```

Comment:
Here, the style is different from your first pipeline. The first one is quoted while these are marked as monospace. So please decide on a consistent format, e.g.

```suggestion
- "measure" : performs the actual time measurement by executing experiments for each of the previously generated configuration with the ability to parallelize processing
- "ctmeasure" : continous a previous time measurement
- "combine" : gathers all the results from the `measure` pipeline and combines them into a single output dataset
- "visualize" : takes the combined experiment results and generates some nice plots
```